igvm/host.py

```diff
@@ -138,13 +139,13 @@ def put(self, remote_path, local_path, mode='0644'):
             )
 
     def acquire_lock(self, allow_fail=False):
-        if self.dataset_obj['igvm_locked']:
+        if bool(self.dataset_obj['igvm_locked']):
```

Comment:
This would be cast to bool implicitly anyhow. If you want to make it more explicit better check for `not None`